README.md

```diff
@@ -127,111 +69,65 @@ To get a local copy up and running, follow these steps.
 
 In order to run this project you need:
 
-<!--
-Example command:
-
-```sh
- gem install rails
-```
- -->
 
 ### Setup
 
 Clone this repository to your desired folder:
 
-<!--
-Example commands:
 
-```sh
-  cd my-folder
-  git clone git@github.com:myaccount/my-project.git
-```
---->
 
-### Install
 
-Install this project with:
+If you want to clone this repository, cd into your desire folder and enter "git clone https://github.com/mumbereh/Hello-world.git into your terminal
+ 
 
```

Comment:
- [ ] To make your documentation more descriptive and professional, I think it would be better if you add some important missing information such as `how to access the cloned directory`, `prerequisites tools needed by the user`, `how to install dependencies(if exist)`, etc. To help you, I suggest you refer this **[template](https://gist.github.com/tresorsawasawa/36552594cf13b97ffd7366e81b09dcfd)**